src/constants/path.ts

```diff
@@ -20,6 +20,9 @@ export const PATH = {
   TODAY_TALKPICK: 'todaytalkpick',
   TALKPICK_PLACE: 'talkpickplace',
   BALANCEGAME: (setId: number | string = ':setId') => `balancegame/${setId}`,
+  BALANCEGAME_EDIT_ROUTE: 'balancegame/edit/:gameSetId',
+  BALANCEGAME_EDIT_LINK: (gameSetId: number | string) =>
+    `balancegame/edit/${gameSetId}`,
```

Comment:
위의 BALANCEGAME처럼 한번에 선언이 가능한데 두 가지로 분리하신 이유가 궁금합니다!

Reply:
경로 변경이나 유지보수를 할 때 좀 더 용이하게 관리하기 위해서 해당 방식을 시도해보았는데, 크게 복잡한 정도의 라우팅 처리는 아니기 때문에 BALANCEGAME 관련 라우팅은 
```suggestion
export const BALANCEGAME = {
  EDIT: 'balancegame/edit/:gameSetId',
  CREATE: 'balancegame/create',
};
```
이런 식으로 그룹핑하는게 현재 규모에서는 더 적합해 보이네요!

Reply:
추가로, 정적 값과 동적 값을 동시에 사용한 이유는
`동적 값: 실제 navigate 로직에서 사용`, `정적 값: App.tsx에서 라우팅 설정을 선언하기 위함` 이었습니다!

Reply:
[c0a54ef](https://github.com/JECT-Study/PICK-O-Client/pull/268/commits/c0a54efe9ec9b4c483cdf40c3aafffe16b983fbf) 에서 라우팅 상수 및 라우트 설정, [efbfa97](https://github.com/JECT-Study/PICK-O-Client/pull/268/commits/efbfa97d82fbb896851a3ff465557649a53bf44a) 에서 네비게이션 상수 처리를 하였습니다!